validator/sawtooth_validator/consensus/poet1/validator_registry.py

```diff
@@ -293,7 +411,11 @@ def __init__(self, minfo=None):
         self.update = None
 
         if 'Update' in minfo:
-            self.update = Update(minfo=minfo['Update'])
+            if 'signup_info' in minfo['Update']:
```

Comment:
We need to think about whether its really ok to just switch on the message contents or whether its better to have an exclusive-or on these policies. I think a ledger that allows a mix of white-listed nodes and full poet nodes to intermix might lead to difficulties.